app/base/views.js

```diff
@@ -1,3 +1,28 @@
+var bole = require('bole');
```

Comment:
don't think `bole` is used in this file.

Reply:
You are right, I had some debug logging and removed it, but forgot to remove the `require`